README.md

```diff
@@ -2,47 +2,60 @@
+-   Fails the workflow if code coverage falls below your project’s [quality settings](https://docs.codacy.com/repositories-configure/adjusting-quality-settings/#gates).
 
-## Workflow options
+To upload coverage to Codacy using the GitHub Action with default settings:
 
-Change these options in the workflow `.yml` file to meet your GitHub project needs:
+1.  Set up an API token to allow the GitHub Action to authenticate on Codacy:
 
-| Setting                           | Description                                                                            | Recommended value                     |
-| --------------------------------- | -------------------------------------------------------------------------------------- | ------------------------------------- |
-| `api-token`                       | [Account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) | `${{ secrets.CODACY_API_TOKEN }}`     |
-| `project-token`                   | [Project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) | `${{ secrets.CODACY_PROJECT_TOKEN }}` |
-| `coverage-reports`                | Optional comma-separated list of reports to send                                       | `''`                                  |
-| `language`                        | Optionally force associating a language with your coverage report                      | `''`                                  |
-| `force-coverage-parser`           | Optionally force using a specific coverage report parser                               | `''`                                  |
+    -   **If you're setting up one repository**, [obtain a project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
+    -   **If you're setting up multiple repositories**, [obtain an account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
+
+    > ⚠️ **Never write API tokens to your configuration files** and keep your API tokens well protected, as they grant owner permissions to your projects on Codacy.
+
+2.  [Generate a supported code coverage report](https://docs.codacy.com/coverage-reporter/#generating-coverage) on each push to your repository.
+
+3.  Add the following to a file `.github/workflows/codacy-coverage-reporter.yaml` in your repository. Update the sample code with the name of the environment storing the access token and the paths to the report files generated previously:
```

Comment:
Having two sentences here makes it difficult to understand that "the following" refers to the sample.
Can we make this step similar to Step 3 in https://github.com/codacy/codacy-analysis-cli-action#integration-with-codacy-for-client-side-tools?
I'm not sure what we need to replace/update.

Reply:
Good idea. I'll also remove the reference to the access token, since it's `CODACY_PROJECT_TOKEN` and we discuss it in the point above.